src/main/resources/org/kuali/kfs/gl/spring-gl.xml

```diff
@@ -57,7 +57,8 @@
 				<value>posterJob</value>
 				<value>posterBalancingJob</value>
 				<value>sufficientFundsAccountUpdateJob</value>
-				<value>clearPendingEntriesJob</value>
+                <value>sufficientFundsFullRebuildJob</value>
```

Comment:
whitespace is weirdly off here, just like it was for me on an earlier PR.  Could you try to fix and see if you have powers over whitespace which i don't?